### PR TITLE
Nodedevcontent

### PR DESCRIPTION
**Pull request by iregina**
Content for the node developers section

### CODE REVIEW

**Comment by iregina** [comment]
Checks failed cause there are some dead/broken links which I removed temporarily

File: docs/node-developers/sandbox-node.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+docker logs --follow mina
+```
+
+And stop mina by running.
+
+```
+docker stop mina
+```
+
+You can use the Mina CLI to interact with the sandbox node. The following command opens a shell inside the docker container from where you can issue any of the available [mina commands](/node-operators/cli-reference).
```
Comment (jasongitmail) [inline docs/node-developers/sandbox-node.mdx:46]: Can we change this link text to `cli commands` please? I know it's old content, but it'll make more sense to readers.

**Review by jasongitmail — APPROVED** [review]
Could you add the `A new version of Mina Docs is coming soon! We're currently updating the contents of page.` note box above the H1 of each page please?

Left one small comment. Otherwise LGTM!

File: docs/node-developers/sandbox-node.mdx (hunk 1 above)
Comment (iregina) [reply docs/node-developers/sandbox-node.mdx:46]: Done

**Comment by jasongitmail** [comment]
Mentioned in Slack yesterday. But to ensure it's not lost--we used `note` for those callouts previously. After that, LGTM

**Pull request merged**